src/main/java/com/flightstats/hub/config/server/JettyServer.java

```diff
@@ -123,23 +124,28 @@ private ResourceConfig buildResourceConfig(Injector injector) {
     }
 
     private SslContextFactory getSslContextFactory() throws IOException {
-
         SslContextFactory sslContextFactory = null;
+
         if (appProperties.isAppEncrypted()) {
             log.info("starting hub with ssl!");
-
-            final String keyStorePath = appProperties.getKeyStorePath() + HubHost.getLocalName() + ".jks";
-            log.info("using key store path: {}", keyStorePath);
-
             sslContextFactory = new SslContextFactory();
-            sslContextFactory.setKeyStorePath(keyStorePath);
-
-            String keyStorePasswordPath = appProperties.getKeyStorePasswordPath();
-            URL passwordUrl = new File(keyStorePasswordPath).toURI().toURL();
-            String password = Resources.readLines(passwordUrl, StandardCharsets.UTF_8).get(0);
-            sslContextFactory.setKeyStorePassword(password);
+            sslContextFactory.setKeyStorePath(getKeyStorePath());
```

Comment:
I like the helper methods! 
When I said "short-circuiting" in the other PR I meant using a guard clause. D'oh. The pattern always bounces out of the method immediately if you don't meet criteria applicable to the method. You do end up with two return statements, but the first one at most is an if/log/return and nothing else, ever.
Since that wasn't part of the refactoring here, I was wondering if you have strong feelings against using them. If so, why?

Reply:
whoops, I didn't check my email so I missed the answer in the other PR. Thanks.